Modules/SubModule_Import/xASL_wrp_NII2BIDS_Subject.m

```diff
@@ -46,7 +46,17 @@
     
     % Go through all (ASL) runs
     for iRun = 1:length(listRuns)
-        x = xASL_imp_NII2BIDS_Run(x, bidsPar, studyPar, listRuns, nameSubjectSession, bidsLabel, iRun);
+		% Get the correct studyPar for a specific subject/visit/run
+		runName = listRuns{iRun};
+		iName = find(runName=='_');
+		if isempty(iName)
+			runName = '';
```

Comment:
first run name should always default to "run-1"to avoid conflicts and to easy programming

Reply:
we've discussed this and there's an issue for that...

Reply:
Sure, but now you make the new issue larger ;) Then this occassion should perhaps be added to this new issue?

Reply:
This is the same line/file you talked about previously. There is no other instance to change the code but this one. The issue stays the same.